starter-repos/shared-ci-cd/appctl/prepare.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+prepare-manifests:
+  only:
+    refs:
+    - master
+  stage: prepare-manifests
+  image: gcr.io/cloud-solutions-images/appctl:app-delivery-demo
+  services:
+    - docker:dind
+  variables:
+    DOCKER_HOST: tcp://localhost:2375
+    DOCKER_TLS_CERTDIR: ""
+  script:
+  - |
+    cat > /root/.appctlconfig <<EOF
+    personal_access_tokens:
+    - host: ${CI_SERVER_HOST}
+      token: ${GIT_ACCESS_TOKEN}
+    EOF
+  - mkdir /root/.ssh
+  - |
+    cat > /root/.ssh/id_rsa <<EOF
+    ${MANIFEST_WRITER_KEY}
+    EOF
+  - chmod 400 /root/.ssh/id_rsa
+  - echo "StrictHostKeyChecking no " > /root/.ssh/config
+  - git config --global user.email ${CI_PROJECT_NAME}-ci@${CI_SERVER_HOST}
+  - git config --global user.name ${CI_PROJECT_NAMESPACE}/${CI_PROJECT_NAME}
+  - |
```

Comment:
Shall this `appctl prepare` setup the gcloud default config? considering that appctl calls gcloud and relies on its default value.

Reply:
We discussed offline. `appctl prepare` in the pipeline won't call gcloud so the settings for gcloud is not necessary.